docs/destinations/forwarding-events/snowbridge/configuration/transformations/builtin/jqFilter.md

```diff
@@ -0,0 +1,39 @@
+# jqFilter
+
+:::note
+This transformation was added in version 3.0.0
+:::
+
+```mdx-code-block
+import JQDescriptionSharedBlock from "./reusable/_jqDescription.md"
+
+<JQDescriptionSharedBlock/>
+```
+
+`jqFilter` filters messages based on the output of a jq command which is run against the data. The provided command must return a boolean result. `false` filters the message out, `true` keeps it.
+
+If the provided jq command returns a non-boolean value error, or results in an error, then the message will be considred invalid, and will be sent to the failure target.
+
+This example filters out all data that doesn't have an `app_id` key.
```

Comment:
what example?

Reply:
The example in the configuration below this statement - which is a file hosted in the GH repo

Reply:
yeah that's not obvious from this, the code below is in a different section of the page (there's a header below) so it doesn't seem related to this statement

Reply:
OK - what change should be made to make it clearer?

Reply:
move this line to underneath the heading